mud-contracts/world/src/modules/inventory/systems/EphemeralInventorySystem.sol

```diff
@@ -238,6 +247,14 @@ contract EphemeralInventorySystem is AccessModified, EveSystem {
     InventoryItem memory item,
     uint256 usedCapacity
   ) internal returns (uint256) {
+    // if (ephemeralInventoryOwner != item.owner) {
```

Comment:
any reason why this check is not needed ?

Reply:
I left them out to include only essential parameter requirement checking for this round.. we can turn these on in a future PR!